include/vctr/Containers/VctrBase.h

```diff
@@ -291,6 +310,58 @@ class VctrBase : private Config,
     //==============================================================================
     // Finding elements and manipulating them
     //==============================================================================
+    /** Returns an iterator to the first element that equaled valueToLookFor or end() if none was found */
```

Comment:
I would prefer "element that equals" here and in the descriptions below as the simple past actually implies that it might have changed already, so it sounds a bit confusing to me (whereas "none was found" appears fine to me).